normandy/recipes/tests/test_filters.py

```diff
@@ -206,6 +207,16 @@ def test_throws_error_on_bad_platform(self):
             filter.to_jexl()
 
 
+class TestNegateFilter(FilterTestsBase):
+    def create_basic_filter(self):
+        pref_filter = PrefExistsFilter.create(pref="browser.urlbar.maxRichResults", value=True)
+        return NegateFilter.create(filter=pref_filter)
```

Comment:
This works great as a unit test, but how could users pass this in via JSON? The data for a recipe's filter object needs to be a simple dict, and there isn't anything that is doing the translation from that to an inflated filter.

I expect that the negate filter will need to do some work using (or at least something like) [`filters.from_data`](https://github.com/mozilla/normandy/blob/8849a4ab066c76adc9ff660daeff5cb2effa068c/normandy/recipes/filters.py#L663-L668).

Reply:
Oh, yea, thanks, of course the child filter would need to be passed in as a JSON object.